src/valuedlg.cpp

```diff
@@ -208,7 +212,13 @@ void ValueDlg::UpdateValueBox() {
     }
     else
     {
-        ui->textEdit->setPlainText(value);
+        // Toggling between "setHtml" and "setPlainText" still winds up formatting
```

Comment:
does this problem already happens before your chance?

Reply:
Yes, however it does not happen on the master build, only dev branch. Possibly also related to the Qt update?